tests/model_fields/test_imagefield.py

```diff
@@ -469,3 +470,30 @@ def test_dimensions(self):
         # Dimensions were recalculated, and hence file should have opened.
         self.assertIs(p.mugshot.was_opened, True)
         self.assertIs(p.headshot.was_opened, True)
+
+
+@skipIf(Image is None, "Pillow is required to test ImageField")
+class NoReadTests(ImageFieldTestMixin, TestCase):
+    PersonNoReadImage = PersonNoReadImage
+
```

Comment:
Afaict, there’s no need to copy this pattern from the above tests, because there are no inherited tests to run. Remove it and the `self.` for the class in the tests.

```suggestion
```